lib/matplotlib/tests/test_legend.py

```diff
@@ -515,6 +515,11 @@ def test_text_nohandler_warning():
         ax.legend()
     assert len(record) == 1
 
+    # this should _not_ warn:
+    f, ax = plt.subplots()
+    ax.pcolormesh(np.random.uniform(0, 1, (10, 10)))
+    ax.get_legend_handles_labels()
```

Comment:
Do you need to be recording the warnings on this line?

Reply:
The PR is correct, see https://stackoverflow.com/a/45671804

Reply:
I think you're commenting on an outdated review.